alphaVantage/alphavantage.js

```diff
@@ -0,0 +1,98 @@
+const axios = require("axios");
+const utils = require("./utils");
+
+module.exports = class alphaVantage {
+	constructor(key) {
+		this.key = key;
+	}
+
+	getStockQuote = async (symbol) => {
+		let uri = `https://www.alphavantage.co/query?function=GLOBAL_QUOTE&symbol=${symbol}&apikey=${this.key}`;
+		let data = await axios
+			.get(uri)
+			.then((res) => {
+				return res.data["Global Quote"];
+			})
+			.catch((err) => {
+				console.log(err);
+				return {};
+			});
+
+		if ("05. price" in data) {
+			const price = data["05. price"];
+			return price;
+		} else {
+			return 0;
+		}
+	};
+
+	getStockInfo = async (keyword) => {
+		let uri = `https://www.alphavantage.co/query?function=SYMBOL_SEARCH&keywords=${keyword}&apikey=${this.key}`;
+		let data = await axios
+			.get(uri)
+			.then((res) => {
+				return res.data["bestMatches"][0];
+			})
+			.catch((err) => {
+				console.log(err);
+				return {};
+			});
```

Comment:
~~Woah, didn't know you could select multiple lines, should've done that earlier~~.

I would use `try`/`catch` and another `await` instead of chaining `.then`.

Reply:
Actually, now that I think about it, you would have to rebind data (which might look messier?), so I'm gonna resolve this.